### PR TITLE
Add a Quick Start page to the Learning/UI docs

### PR DESCRIPTION
**Pull request by NancyMcCallB**
Added docs to help new developers get an app started using the iTwinViewer template and dynamic UI  modification.
Added the "npm install" step to the create-react-app instructions.

### CODE REVIEW

File: docs/learning/tutorials/develop-web-viewer.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites).
 - From a terminal, `npx create-react-app your-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`.
   - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- In the `your-app-name` directory, `npm install`.
```
Comment (calebmshafer) [inline docs/learning/tutorials/develop-web-viewer.md:8]: @NancyMcCallB you shouldn't need to call `npm install` here.  The create-react-app generator runs installation for you.

**Review by pmconne — APPROVED** [review]

File: docs/learning/ui/AugmentingUI.md [hunk 2]
```diff
@@ -34,41 +66,9 @@ export class MyUiItemProvider {
 Register the UiItemsProvider.
 
 ```ts
-UiItemsManager.register(new MyUiItemProvider(IModelApp.i18n));
+UiItemsManager.register(new MyUiItemProvider(iTwinApp.i18n));
```
Comment (calebmshafer) [inline docs/learning/ui/AugmentingUI.md:69]: This instance of IModelApp shouldn't be changed.

Comment (NancyMcCallB) [reply docs/learning/ui/AugmentingUI.md]: Oops! I thought I bypassed all the IModelApp instances.

File: docs/learning/tutorials/develop-web-viewer.md (hunk 1 above)
Comment (NancyMcCallB) [reply docs/learning/tutorials/develop-web-viewer.md:8]: Yep. I got an error that npm install fixed. I just did a new create-react-app, though, and it was fine. I'll remove it.

**Review by calebmshafer — APPROVED** [review]

File: docs/learning/ui/AugmentingUI.md (hunk 2 above)
Comment (NancyMcCallB) [reply docs/learning/ui/AugmentingUI.md]: I take it  back: this is what I get when I do an npm start without installing:

Failed to compile.

./node_modules/@microsoft/applicationinsights-web/dist-esm/applicationinsights-web.js
Attempted import error: 'PerfEvent' is not exported from '@microsoft/applicationinsights-core-js'.

File: docs/learning/tutorials/develop-web-viewer.md (hunk 1 above)
Comment (NancyMcCallB) [reply docs/learning/tutorials/develop-web-viewer.md]: Failed to compile.

./node_modules/@microsoft/applicationinsights-web/dist-esm/applicationinsights-web.js
Attempted import error: 'PerfEvent' is not exported from '@microsoft/applicationinsights-core-js'.

File: docs/learning/ui/AugmentingUI.md (hunk 2 above)
Comment (NancyMcCallB) [reply docs/learning/ui/AugmentingUI.md]: Oops! I thought I skipped all the IModelApp instances in code.

File: docs/learning/tutorials/develop-web-viewer.md (hunk 1 above)
Comment (calebmshafer) [reply docs/learning/tutorials/develop-web-viewer.md]: What version of Node are you using?  I'll give it a test now.

Comment (calebmshafer) [reply docs/learning/tutorials/develop-web-viewer.md]: Also can you try doing `--use-npm`?

**Pull request merged**